bots/images/scripts/lib/kubevirt.setup

```diff
@@ -1,48 +1,67 @@
 #!/bin/bash
-set -ex
 
-function docker_pull {
-  docker pull $1
-  echo "$1" >> /tmp/pulledImages
+set -x
```

Comment:
I really don't like dropping the `set -e`. This makes it much much harder to debug stuff. (Something for a follow-up)